app/src/main/res/layout/activity_calendar.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file should probably be renamed to `activity_main` to keep with the java file name.

Reply:
good catch, I thought refactor would have taken care of it.